src/encoded/schemas/publication.json

```diff
@@ -37,7 +38,18 @@
         },
         "date_published": {
             "title": "Publication date",
-            "type": "string"
+            "description": "The date the publication or communication was published; must be in YYYY-MM-DD format.",
+            "type": "string",
+            "oneOf": [
+                {
+                    "format": "date"
+                },
+                {
+                    "comment": "Historical date formats, not valid for new submissions.",
+                    "permission": "import_items",
+                    "pattern": "^[0-9]{4}[^-]*$"
```

Comment:
The regex previously required a 5th character after the year, updated here to allow just the year in import_items